src/Exceptions/SmartlingApiException.php

```diff
@@ -123,7 +123,12 @@ public function formatErrors($title = '')
                 $errorsStr,
             ]
         );
-        
+
```

Comment:
This change allows access to full response body, it is truncated now, making it difficult to work with exceptions

Reply:
Could you please provide an example `before/after`? I didn't get the problem

Reply:
Before:
```
::: Rename File Example :::
Error happened while renaming files
Response code: 0
Response errors (0): 
---------------------------


```
After, a line is added:

```
GuzzleHttp\Exception\ClientException: {"response":{"code":"VALIDATION_ERROR","errors":[{"key":"file.not.found","message":"The file \"test.xml\" could not be found","details":{"field":"fileUri"}}]}}
```
formatErrors is used only in our examples, but I think having the response details from API is beneficial